css/search.css

```diff
@@ -8,10 +8,6 @@ li.job-card-container__applicant-count,
   display: none !important;
 }
 
-span.tvm__text:not(:has(span)) {
```

Comment:
I removed this - it was hiding extra text that met the same requirements (a parent `span` element with class `.tvm__text` with a child element that does not have a child `span` element).